themes/kiali/static/css/kiali-home.css

```diff
@@ -76,16 +78,14 @@ div#features-blocks a:hover div {
 }
 
 div#features-blocks div {
-  height: 210px;
-  width: 290px;
+  height: 150px;
+  width: 420px;
```

Comment:
These new width are not mobile friendly.
If you want this width for desktop it's OK. But for mobile screens the feature cards need to adapt to the width of the screen.